driver/src/main/java/org/neo4j/driver/v1/Config.java

```diff
@@ -256,30 +261,47 @@ public ConfigBuilder withMaxIdleSessions( int size )
         }
 
         /**
-         * Pooled sessions that have been unused for longer than this timeout
-         * will be tested before they are used again, to ensure they are still live.
+         * Please use {@link #withConnectionLivenessCheckTimeout(long, TimeUnit)}.
          *
+         * @param timeout minimum idle time in milliseconds
+         * @return this builder
+         * @see #withConnectionLivenessCheckTimeout(long, TimeUnit)
+         * @deprecated please use {@link #withConnectionLivenessCheckTimeout(long, TimeUnit)} method. This method
+         * will be removed in future release.
+         */
+        @Deprecated
+        public ConfigBuilder withSessionLivenessCheckTimeout( long timeout )
+        {
+            withConnectionLivenessCheckTimeout( timeout, TimeUnit.MILLISECONDS );
+            return this;
+        }
+
+        /**
+         * Pooled sessions that have been idle in the pool for longer than this timeout
```

Comment:
Pls use `Pooled connections` instead as `pooled sessions` is a wrong terminology used before.